force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,18 +197,13 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
-
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        String hw = '';
+        for(Integer i = 1; i <= repeatCount; i++){
+            hw += inputString; 
+        if(i < repeatCount) {
+            hw += '; '; }
```

Comment:
Good job. Take a look at the Sring methods. Repeat and removeEnd. You can save yourself some work. 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_string.htm-